feature/player/src/main/java/dev/anilbeesetti/nextplayer/feature/player/PlayerActivity.kt

```diff
@@ -917,6 +942,34 @@ class PlayerActivity : AppCompatActivity() {
         exoContentFrameLayout.requestLayout()
     }
 
+    private fun setVideoLoop(videoLoop: VideoLoop, showInfo: Boolean) {
+        viewModel.setVideoLoop(videoLoop)
+        when (videoLoop) {
+            VideoLoop.LOOP_OFF -> {
+                player.repeatMode = Player.REPEAT_MODE_OFF
+                loopVideoButton.setImageResource(coreUiR.drawable.ic_repeat_off)
+            }
+
+            VideoLoop.LOOP_ONE -> {
+                player.repeatMode = Player.REPEAT_MODE_ONE
+                loopVideoButton.setImageResource(coreUiR.drawable.ic_repeat_one)
+            }
+
+            VideoLoop.LOOP_ALL -> {
+                player.repeatMode = Player.REPEAT_MODE_OFF
```

Comment:
```suggestion
            VideoLoop.LOOP_ALL -> {
                player.repeatMode = Player.REPEAT_MODE_ALL
```

Reply:
Player.REPEAT_MODE_ALL will keep playing the same video infinitely; it acts the same as Player.REPEAT_MODE_ONE because, if you try to print the mediaItemCount, it returns 1, which is the currently playing media item.

In the playVideo() method, this line specifies that only one media item is added in the player's mediaItems list and not the whole playlist.
` player.setMediaItem(mediaStream, viewModel.currentPlaybackPosition ?: C.TIME_UNSET)`


Player.REPEAT_MODE_ALL  would have worked if  `player.setMediaItems(mediaStreamList)` would have been used instead of `player.setMediaItem(mediaStream)`, i.e. , adding the entire playlist at once and not clearing up the player's mediaItems list and then adding a single mediaItem every time playVideo() is called.

Reply:
Got it